app/controllers/internal/configs_controller.rb

```diff
@@ -10,7 +10,11 @@ def show
   def create
     clean_up_params
     config_params.keys.each do |key|
-      SiteConfig.public_send("#{key}=", config_params[key].strip) unless config_params[key].nil?
+      if config_params[key].respond_to?(:to_h)
```

Comment:
array, structs and others respond to `.to_h`. maybe we should simply check if the value `.is_a?(Hash)` ?

Reply:
@rhymes It's not a hash at that point though, we only turn it into a hash in the line below. We could do check is_a?(ActionController::Parameters) as that's what it is at that point.

Reply:
ooooh cool :)